zulipterminal/ui_tools/boxes.py

```diff
@@ -1516,13 +1520,13 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
                 if self.message['type'] == 'private':
                     self.model.controller.view.set_footer_text(
                             " Time Limit for editing the message has"
-                            " been exceeded.", 3)
+                            " been exceeded.", 'task:failure', 3)
                     return key
                 elif self.message['type'] == 'stream':
                     self.model.controller.view.set_footer_text(
                             " Only topic editing allowed."
                             " Time Limit for editing the message body has"
-                            " been exceeded.", 3)
+                            " been exceeded.", 'task:failure', 3)
```

Comment:
This seems like a good candidate for `task:warning`?
As in, the user is still allowed to edit, but not the message body.